nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/MergeRecord.java

```diff
@@ -282,29 +288,89 @@ public final void resetState() {
         }
         if (minRecords != null && minRecords <= 0) {
             results.add(new ValidationResult.Builder()
-                    .subject("Min Records")
-                    .input(String.valueOf(minRecords))
-                    .valid(false)
-                    .explanation("<Minimum Number of Records> property cannot be negative or zero")
-                    .build());
+                .subject("Min Records")
```

Comment:
Note: The original indentation was weird with respect to the other validation cases